ui/main/src/app/modules/cards/components/detail/detail.component.ts

```diff
@@ -131,7 +138,13 @@ export class DetailComponent implements OnChanges, OnInit, OnDestroy, AfterViewC
 
     ngOnInit() {
 
-        if (this._appService.pageType === PageType.FEED) {
+        this.lttdExpired$ = this.store.select(selectLttdExpired);
+        this.subscription1$ = this.lttdExpired$.subscribe((result) => {
```

Comment:
Add unsubscribe when destroying the object

Reply:
I factor the code, I don't need this.lttdExpired$  anymore.